mirrulations-work-generator/src/mirrgen/work_generator.py

```diff
@@ -55,9 +55,15 @@ def generate_work():
 
         # Download dockets, documents, and comments
         # from all jobs in the job queue
+        print('Begin generate docket jobs')
         generator.download('dockets')
+        print('End generate docket jobs')
+        print('Begin generate document jobs')
         generator.download('documents')
+        print('End generate document jobs')
+        print('Begin generate comment/attachment jobs')
         generator.download('comments')
+        print('End generate commend/attachment jobs')
```

Comment:
Quick spelling error here. Can you change this to comment?

Reply:
done